README.md

```diff
@@ -27,31 +29,109 @@ npm install @dojo/widgets
+
+- All widgets support internationalisation (`i18n`)
+
+## Widgets
+
+### Form widgets
+[Button](src/button/README.md)
+
+[Checkbox/Switch](src/checkbox/README.md)
+
+[ComboBox](src/combobox/README.md)
+
+[Label](src/label/README.md)
+
+[Radio](src/radio/README.md)
+
+[Select](src/select/README.md)
+
+[Slider](src/slider/README.md)
 
-Dojo 2 widgets are fully accessible (`a11y`), out of the box.
+[TextArea](src/TextArea/README.md)
 
-Dojo 2 widgets are fully themeable. In a future release we will ship with several official Dojo 2 themes that can be applied to the widgets (all themes will live in a separate repo).
+[TextInput](src/TextInput/README.md)
 
-Dojo 2 widgets are fully internationalisable (`i18n`)
+### Layout widgets
+[SlidePane](src/slidepane/README.md)
+
+[TabPane](src/tabpane/README.md)
+
+### Misc widgets
+[Dialog](src/dialog/README.md)
 
 ## Conventions
 
+### EventHandlers
+
+You can register event handlers that get called when the corresponding events occur, by passing the handlers into a widget, in their `properties`.
+The naming convention for event handlers is as follows:
+
+- if the parent of the widget has the power to decide *if* an event is successful, i.e. can cancel the event, then the child widget will call an event handler in the following format:
+
+`onRequest[X]`, e.g. for a `close` event, the event handler called by the child widget must be called `onRequestClose`
+
+- for events that will occur regardless of child/parent interaction, then the `Request` naming convention is dropped:
+
+`on[X]`, e.g. for a `dismiss` event, then event handler called by the child widget must be called `onDismiss`
+
+### Controlled vs uncontrolled form widgets
+Currently, all of our *form* widgets are 'controlled' widgets. This means that a parent widget is responsible to the 'state' of the child widget.
+'Controlled' widgets are also known as 'stateless' widgets.
+
 ### Icons
 
 We use [font awesome](http://fontawesome.io/) for icons.
 Where a theme requires specific icons that are not part of the Font Awesome set, then those themes will ship their own icons.
 
 ### Coding conventions
 
-`px vs em` - we specify font sizes in `px`. When creating a widget, spacing (margin, padding) should be specified using `px` unless the design calls for proportional spacing, in which case `em` can be used.
+`px vs em` - we specify font sizes in `px`.
+When creating a widget, spacing (margin, padding) should be specified using `px` unless the design calls for proportional spacing, in which case `em` can be used.
+
+
+## How to customise a widget
+
+There are may ways in which you can customise the behaviour and appearance of our widgets.
+See the `widget-core` [README](https://github.com/dojo/widget-core) for examples of how to customise the theme or a specific CSS class of a widget.
 
+Or can you write your own widget that extends an official widget.
```

Comment:
What's an official widget (meaning, let's not introduce new terminology)? So perhaps Dojo widget, or an `@dojo/widgets` widget or provided/included widgets?